xcell/mappers/mapper_Planck_base.py

```diff
@@ -39,23 +45,20 @@ def get_mask(self):
         if self.mask is None:
             if self.file_mask is not None:
                 self.mask = hp.read_map(self.file_mask)
-                self.mask = hp.ud_grade(self.mask,
-                                        nside_out=self.nside)
             else:
                 self.mask = np.ones(12*self.nside**2)
```

Comment:
Since you deleted the `ud_grade` above, this should not be `12*self.nside**2`...